### PR TITLE
Add re-exports

### PR DESCRIPTION
**Pull request by tomjaguarpaw**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # ChangeLog for typed-process
 
+## 0.2.7.1
```
Comment (snoyberg) [inline ChangeLog.md:3]: I think following PVP this should be 0.2.8.

Comment (tomjaguarpaw) [reply ChangeLog.md]: Ah, thank you. Fixed!

**Comment by tomjaguarpaw** [comment]
Something seems to be wrong with the MacOS CI.

**Review by snoyberg — APPROVED** [review]
LGTM! Since we’re still getting used to joint maintenance, do you want to handle merge abs release? Or something else?

**Comment by tomjaguarpaw** [comment]
> do you want to handle merge abs release?

Yes sure, happy to.

**Comment by tomjaguarpaw** [comment]
Released to Hackage https://hackage.haskell.org/package/typed-process-0.2.8.0

**Pull request merged**